docs/Concepts/Qubernetes/Multicluster.md

```diff
@@ -1,69 +1,76 @@
-## Summary
-
-As GoQuorum networks grow and need HA and to be managed between different business units / companies, etc. we need to have a
-way to connect GoQuorum nodes across multiple organizations, availability zones, etc. As the industry moves towards
-Kuberenetes, we need a way to support this cross cluster model in Kubernetes. This document outlines the initial design
-for Qubernetes to manage and enable a cross-cluster GoQuorum Kubernetes deployment.
+---
+description: Qubernetes for multi-cluster networks
+---
 
+# Qubernetes for multi-cluster networks
 
+Multi-cluster networks support high availability and connect different business units
+and companies. The following outlines using Qubernetes to manage and enable a cross-cluster GoQuorum
```

Comment:
maybe add "The following outlines an initial way to use Qubernetes to manage and enable"

Reply:
Done